frontend/public/components/persistent-volume-claim.jsx

```diff
@@ -127,8 +127,8 @@ const PVCTableRow = ({ obj, index, key, style }) => {
         {obj?.spec?.storageClassName ? (
           <ResourceLink
             kind="StorageClass"
-            name={obj?.spec?.storageClassName}
-            title={obj?.spec?.storageClassName}
+            name={obj.spec.storageClassName}
+            title={obj.spec.storageClassName}
           />
         ) : (
           <div className="text-muted">-</div>
```

Comment:
Can you remove the `text-muted` here while in the file? It's inconsistent with other `-` cells in the same table that don't have the `text-muted` style.

Reply:
> Can you remove the `text-muted` here while in the file? It's inconsistent with other `-` cells in the same table that don't have the `text-muted` style.

Done. Thanks.